### ISSUE TITLE
Impossible to make barcharts (horizontal columns)

### ISSUE THREAD

**Issue by georgegeorge88** [pablojim/highcharts-ng#580]
## Link to jsfiddle showing the issue.

https://rawgit.com/pablojim/highcharts-ng/master/example/charts/general-example.html

## Expected Behaviour

If I change the default type to bar or add just bar series I expect horizontal bars. If I copy the same chart config to the basic demo, it will make horizontal bars just fine.

## Current Behaviour

Always column chart (vertical bars) instead of bar chart (horizontal bars).

## Your Environment

* Version of highcharts-ng used: 1.1
* Browser Name and version: Chrome
* Operating System and version (desktop or mobile): windows 10

I have the same problem in my own angular project, but it happens in this general demo of highcharts-ng too.

**Comment by pablojim** [comment]
Thanks for reporting this. Could you please clarify the issue a little. Is the problem that when you initially have a column series that you can't update them to bar series? Or you can't create a chart with bar series at all? 

Maybe jsfiddle with clear steps to reproduce would help

**Comment by scatcher** [comment]
@pablojim If you go to your example [here](https://rawgit.com/pablojim/highcharts-ng/master/example/charts/general-example.html) and change from column to bar you'll see that they both render as vertical bars instead of the horizontal bars you'd expect in a bar chart.  Let me know if you have any issues recreating.

**Comment by georgegeorge88** [comment]
It seems that now that I did the 0.0.13 -> 1.1.0 update again on my site, it just works, so maybe it was totally something else or a highcharts bug.

And for the problem in the demo. It is a highcharts "feature". You can update chart/series type with .update, BUT it will not work "right" for change between column and bar chart (orientation doesn't update). You can sort of fake it updating the inverted value, but it would be better to just reinitialize the graph. So I think to make your demo work right you should just remove bar from the type options. Or you would have to make some logic to check for change between bar and column and reinitialize the chart in that case.

TLDR you can just close this.

**Comment by georgegeorge88** [comment]
@pablojim Well. I don't know if I should close this myself.

**Comment by pablojim** [comment]
Thanks @georgegeorge88 - fine for you to close it yourself!
